oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -142,44 +134,58 @@ public static ElasticBulkProcessorHandler getBulkProcessorHandler(@NotNull Elast
         return new ElasticBulkProcessorHandler(elasticConnection, indexName, indexDefinition, definitionBuilder, waitForESAcknowledgement);
     }
 
-    private BulkProcessor initBulkProcessor() {
-        return BulkProcessor.builder(requestConsumer(),
-                new OakBulkProcessorListener(), this.indexName + "-bulk-processor")
-                .setBulkActions(indexDefinition.bulkActions)
-                .setConcurrentRequests(BULK_PROCESSOR_CONCURRENCY)
-                .setBulkSize(new ByteSizeValue(indexDefinition.bulkSizeBytes))
-                .setFlushInterval(TimeValue.timeValueMillis(indexDefinition.bulkFlushIntervalMs))
-                .setBackoffPolicy(BackoffPolicy.exponentialBackoff(
-                        TimeValue.timeValueMillis(indexDefinition.bulkRetriesBackoff), indexDefinition.bulkRetries)
-                )
-                .build();
+    private BulkIngester<String> initBulkIngester() {
+        // BulkIngester does not support retry policies. Some retries though are already implemented in the transport layer.
```

Comment:
Since indexDefinition.bulkRetriesBackoff is not longer being used, should we remove the configuration from code ?

Reply:
Makes sense. I have removed them. I think bulk retries should be made configurable at ElasticConnection level. This will be addressed in a separate PR.